effect_envelope.cpp

```diff
@@ -125,24 +131,24 @@ void AudioEffectEnvelope::update(void)
 		sample78 = *p++;
 		p -= 4;
 		mult += inc;
-		tmp1 = signed_multiply_32x16b(mult, sample12);
+		tmp1 = signed_multiply_32x16b(mult >> ENVELOPE_ALIASING, sample12);
```

Comment:
Maybe initialize a variable here, so (mult>>ENVELOPE_ALIASING) doesn't have to happen more than once.  (Unless the compiler already did that.)